src/main/include/Constants.h

```diff
@@ -44,12 +55,26 @@ namespace CONSTANTS
 
     }
 
+    namespace BELTIVATOR
+    {
+        constexpr units::angle::turn_t BOTTOM_POS = 0_tr;
+        constexpr units::angle::turn_t TOP_POS    = 0_tr;
+        constexpr units::angle::turn_t THRESHOLD  = 0_tr;
+        constexpr int BELTIVATOR_ID               = 99; //CHANGEME
```

Comment:
CAN IDs max out at 64, I get that this is a placeholder, but you should keep placeholder CAN IDs within valid ranges